src/libs/state.ts

```diff
@@ -4,12 +4,72 @@ import type { WorkspaceWrapper } from 'src/libs/workspace-utils';
 
 export const routeHandlersStore = atom<unknown[]>([]);
 
-export const authStore = atom<any>({
+export type TerraUser = {
+  token?: string | undefined;
+  scope?: string | undefined;
+  id?: string | undefined;
+  email?: string | undefined;
+  name?: string | undefined;
+  givenName?: string | undefined;
+  familyName?: string | undefined;
+  imageUrl?: string | undefined;
+  idp?: string | undefined;
+};
+
+export type TerraUserProfile = {
+  institute?: string | undefined;
+  title?: string | undefined;
+  department?: string | undefined;
+  interestInTerra?: string | undefined;
+};
+
+export type TokenMetadata = {
+  // do not log or send this to a user (could move to hash in the future)
```

Comment:
Was this intended to be "or send this to Mixpanel"? A user will have access to their own token.

Reply:
Yeah, I will make it more clear. Just not to log or send to mixpanel